fighters/robot/src/opff.rs

```diff
@@ -18,6 +18,35 @@ unsafe fn gyro_dash_cancel(boma: &mut BattleObjectModuleAccessor, status_kind: i
     }
 }
 
+unsafe fn uspecial_cancels(boma: &mut BattleObjectModuleAccessor, situation_kind: i32, frame: f32) {
+    if WorkModule::is_flag(boma, *FIGHTER_ROBOT_STATUS_BURNER_FLAG_PUSH_B_BUTTON) {
+        if situation_kind == *SITUATION_KIND_AIR {
+            if frame > 11.0 {
+                if boma.is_button_on(Buttons::Guard) {
+                    WorkModule::unable_transition_term_group(boma, *FIGHTER_STATUS_TRANSITION_GROUP_CHK_AIR_ESCAPE);
+                    ControlModule::clear_command_one(boma, *FIGHTER_PAD_COMMAND_CATEGORY1, *FIGHTER_PAD_CMD_CAT1_AIR_ESCAPE);
+                    StatusModule::change_status_request_from_script(boma, *FIGHTER_STATUS_KIND_FALL, false);
+                    let vec = Vector3f{x: 0.0, y: -0.2, z: 0.0};
+                    KineticModule::add_speed(boma, &vec);
+                } else if boma.is_button_on(Buttons::Attack) {
+                    StatusModule::change_status_request_from_script(boma, *FIGHTER_ROBOT_STATUS_KIND_SPECIAL_HI_ATTACK, false);
+                }
+            }
+        }
+	}
+}
+
+// JC grounded sideb on hit
+unsafe fn jc_sideb(boma: &mut BattleObjectModuleAccessor, cat1: i32, status_kind: i32, situation_kind: i32, motion_kind: u64) {
+    if (status_kind == *FIGHTER_STATUS_KIND_SPECIAL_S || status_kind == *FIGHTER_ROBOT_STATUS_KIND_SPECIAL_S_END || status_kind == *FIGHTER_ROBOT_STATUS_KIND_SPECIAL_S_ATTACK) {
+        if (AttackModule::is_infliction_status(boma, *COLLISION_KIND_MASK_HIT) && !boma.is_in_hitlag()) {
+            if ControlModule::check_button_on(boma, *CONTROL_PAD_BUTTON_JUMP) {
+                boma.check_jump_cancel(false);
+            }
+        }   
+    }
+}
```

Comment:
Please avoid nesting `if` blocks, and instead use [guard clauses](https://deviq.com/design-patterns/guard-clause) or [logical-and](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) to make conditional blocks easier to read and maintain.